Spec/RealtimeClientConnection.swift

```diff
@@ -2,7 +2,7 @@
 //  RealtimeClient.connection.swift
 //  ably
 //
-//  Created by Ricardo Pereira on 03/11/2015.
```

Comment:
Oops, automation, grumble grumble, etc..

I shall push a commit up to fix this.

Reply:
https://github.com/ably/ably-cocoa/pull/1081/commits/643d4aee4ad6c3ee57c81c7cdc9db2ea788c345c